recipes/grpc/all/patches/v1.37.x/0002-fix-gcc-abseil-link.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 9d48e327c..6af67edfc 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -5,3 +5,6 @@ include(conanbuildinfo.cmake)
+ conan_basic_setup(KEEP_RPATHS)
+ 
+ add_subdirectory("source_subfolder")
++
++# TODO: move to a patch? It avoids link errors while resolving abseil symbols with gcc
++set_target_properties(check_epollexclusive PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
I meant patch upstream CMakeLists, because this patch will break while moving recipes to CMakeToolchain.

Reply:
Why would you want to download each version? Just clone grpc repo, and for each version: checkout, fix CMakeLists (1 line), git diff and put each diff in a patch. There is no other solution for maintainability, otherwise you delegate this task to the poor contributor who will have to upgrade this recipe for conan v2.